data/library/tlg0059/tlg003/metadata.json

```diff
@@ -11,24 +11,23 @@
   ],
   "versions": [
     {
-      "urn": "urn:cts:greekLit:tlg0059.tlg003.jkt:",
+      "urn": "urn:cts:greekLit:tlg0059.tlg003.jkt-1:",
       "node_kind": "version",
       "version_kind": "edition",
-      "first_passage_urn": "urn:cts:greekLit:tlg0059.tlg003.jkt:001-009",
+      "first_passage_urn": "urn:cts:greekLit:tlg0059.tlg003.jkt-1:001-009",
```

Comment:
I think I got this wrong when I did it, I went from the first milestone `{43a}` to the next `{43a}` but that's not right in terms of subsequent pagination (ie `001-009` is only a meaningful size between those particular milestones) so I think this should probably just be: `urn:cts:greekLit:tlg0059.tlg003.jkt:001`?

Reply:
`001` might be too small for what we'd initial want to display for the work.

I'd propose `001-006` as a slightly better chunk than `001-009` but not too short.